assets/public/robots.txt

```diff
@@ -0,0 +1,13 @@
+User-agent: *
+Disallow: */old-*
+Disallow: */referrals*
+Disallow: */forever*
+Disallow: */new-member/*
+Disallow: */welcome?code=
+Disallow: */partner*
+Allow: */partner-program
```

Comment:
Why do we need to specifically allow this path? 🙃 Aren't all other paths crawled by default?

Reply:
It's because it's under `/*/partner/partner-program/*`